arcgis-ios-sdk-samples/Cloud & Portal/Integrated Windows Authentication/README.md

```diff
@@ -1,33 +1,45 @@
 # Integrated Windows Authentication
 
-Uses Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+Connect to an IWA secured Portal and search for maps.
 
-![Screenshot of Integrated Windows Authentication sample](image1.png)
+![Integrated Windows Authentication sample](integrated-windows-authentication.png)
 
 ## Use case
 
-IWA, which is built into Microsoft Internet Information Server (IIS), works well for intranet applications, but isn't always practical for internet apps.
+Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications but isn't always practical for internet apps.
 
 ## How to use the sample
 
-1. Enter the URL to your IWA-secured portal in the text field above the Search Private button.
+1. Enter the URL to your IWA-secured portal in the text field above the Search Secure button.
```

Comment:
👍🏻 That was a last minute change and it looks like I forgot to update the screenshot as well. 😞

Reply:
Didn't notice the screenshot! I'll update it.